src/components/item/sharing/CsvInputParser.tsx

```diff
@@ -43,46 +46,46 @@ const CsvInputParser = ({ item }: Props): JSX.Element => {
   const { t: translateBuilder } = useBuilderTranslation();
   const { t: translateMessages } = useMessagesTranslation();
   const { t: translateCommon } = useCommonTranslation();
-  const { id: itemId, path: itemPath } = item;
+  // const { id: itemId, path: itemPath } = item;
+  const { id: itemId } = item;
   const [isOpen, setIsOpen] = useState(false);
   const {
-    mutate: share,
+    // mutate: share,
     isLoading,
     isSuccess,
     isError,
     data: results,
     error,
   } = mutations.useShareItem();
 
+  // const { mutate: postItem } = mutations.usePostItem();
   const openModal = () => {
     setIsOpen(true);
   };
+
+  const { openModal: openMoveModal, selId: idTemplate, cleanItemSel: cleanId } = useContext(SelectItemModalContext);
+
   const handleClose = () => {
+    if (cleanId)
+      cleanId();
     setIsOpen(false);
   };
 
+  const { data: itemObj } = useItem(idTemplate)
   const handleFileChange = (e: ChangeEvent<HTMLInputElement>) => {
     const t = e.target as HTMLInputElement;
     if (t.files?.length) {
       const file = t.files?.[0];
-
       if (file) {
-        Papa.parse(file, {
-          header: true,
-          dynamicTyping: true,
-          complete: ({ data: parsedData }) => {
-            // add current item path and default permission read
-            const dataWithItemPath = parsedData.map<Partial<Invitation>>(
-              (d) => ({
-                permission: PermissionLevel.Read,
-                ...(d as Partial<Invitation>),
-                itemPath,
-              }),
-            ) as any;
-
-            share({ data: dataWithItemPath, itemId });
+        const formData = new FormData();
+        formData.append("file", file);
+        // TO-DO: this request needs to be moved to query-client
+        axios.post(`${API_HOST}/items/${itemId}/invitations/upload_csv?id=${itemId}&template_id=${idTemplate}`, formData, {
```

Comment:
You need to do something with the answer to this request. You should provide the user with a feedback of the operation.